### DISCUSSION TITLE
so-tcpreplay fails for 2.3.21

### DISCUSSION THREAD

**Discussion by jpl33** [Security-Onion-Solutions/securityonion#2453] [General] [answered]
Command 'docker pull --disable-content-trust=false docker.io/securityonion/so-tcpreplay:2.3.21' failed with return code: 1
return message is : 'No valid trust data for 2.3.21'
but 
command ' sudo docker pull --disable-content-trust=false docker.io/securityonion/so-tcpreplay:2.3.10'
succeeds and downloads the 2.3.10 docker.
seems like some trouble with docker.io?

**Comment by jertel** [comment]
SO Images are no longer hosted on Docker Hub (docker.io) due to their rate limiting they put in place last month. I'm curious why you are trying to pull the images directly. Is soup not working?

**Reply by jpl33** [reply]
I just copied the contents of the script in github and ran them individually :-)
you guys better update the github.

**Comment by jertel** [comment, marked as answer]
Tracking this in issue #2463
